__tests__/app.test.js

```diff
@@ -174,3 +174,54 @@ describe('/api/articles', () => {
         })
     })
 })
+
+describe('/api/articles/:article_id/comments', () => {
+    test('POST 201 - post a comment to an article', () => {
+        const newComment = { username: "icellusedkars", body: "This article is great!" }
+        return request(app)
+        .post('/api/articles/2/comments')
+        .send(newComment)
+        .expect(201)
+        .then(({body}) => {
+            const { comment } = body
+            expect(comment).toMatchObject({
+                comment_id: expect.any(Number),
+                article_id: 2,
+                author: "icellusedkars",
+                body: "This article is great!",
+                votes: 0,
+                created_at: expect.any(String)
+            })
+        })
+    })
+    test('POST 400 - PSQL error: comment malformed', () => {
+        const newComment = { body: "This article is great!" }
+        return request(app)
+        .post('/api/articles/banana/comments')
+        .send(newComment)
+        .expect(400)
+        .then(({body}) => {
+            expect(body.msg).toBe('Bad request')
```

Comment:
Could this test please be separated into two? So we'd have one test where the username is missing, then one where we go to articles/banana/comments, both to expect a 400 👍

Reply:
Separated into a missing body and a article_id = banana